build.gradle

```diff
@@ -74,7 +74,7 @@ allprojects {
         }
         maven {
             name = "NukkitX Snapshots"
-            url = "https://repo.nukkitx.com/maven-snapshots/"
```

Comment:
This is the correct one

Reply:
i was having issues with intellij throwing lots of "not found" errors at me and reverting back to the url i'd been using originally fixed them, but i'll change it back now. thanks!